profiles/models.py

```diff
@@ -279,6 +280,7 @@ class VerifiedPersonalInformation(ValidateOnSaveModel, NullsToEmptyStringsModel)
         max_length=4,
```

Comment:
If it's always exactly 3 digits, could the field have maximum length of 3 too?

Reply:
For consistency that would make sense. I'll fix it.

Reply:
Changed `max_length` to 3.